docs/source/guides/versioning.md

```diff
@@ -5,27 +5,33 @@ description: Everything you need to know about evolving your GraphQL schema
 
 Versioning is a technique to prevent necessary changes from becoming "breaking changes" which affect the existing consumers of an API.  These iterations might be as trivial as renaming a field, or as substantial as refactoring the whole data model.
 
-Developers who have worked with REST APIs in the past have probably recognized various patterns for versioning the API, commonly by using a different URI (e.g. `/api/v1`, `/api/v2`, etc.) or a query parameter (e.g. `?version=1`).
+Developers who have worked with REST APIs in the past have probably recognized various patterns for versioning the API, commonly by using a different URI (e.g. `/api/v1`, `/api/v2`, etc.) or a query parameter (e.g. `?version=1`).  With this technique, an application can easily end up with many different API endpoints over time, and the question of _when_ an API can be deprecated can become problematic.
 
-It’s certainly responsible to consider versioning a GraphQL API in a similar fashion, but GraphQL APIs don't need the same type of versioning and, assuming the correct strategies and precautions are observed, are able to serve various iterations on an API from a single endpoint.
+It might be tempting to version a GraphQL API in the same way, but with the right preparations GraphQL APIs don't require the same versioning technique.  By following the strategies and precautions outlined in this guide and using Apollo tooling that adds clarity to every change, many iterations of an API can be served from a single endpoint.
```

Comment:
> but with the right preparations....

That's weird in the sentence. Can it be worded differently like:

`It might be tempting to version a GraphQL API the same way, but there's a versioning technique that works better for GraphQL APIs than the former. By following....`